.travis.yml

```diff
@@ -9,12 +12,18 @@ python:
   - "2.7"
   - "3.5"
 
+env:
+  - TEST_DIR=root TEST_TYPE=lint
+  - TEST_DIR=root TEST_TYPE=unit
+  - TEST_DIR=root TEST_TYPE=functional
+  - TEST_DIR=utils
+
 install:
   - pip install -r requirements.txt
   - pip install tox-travis
 
 script:
   # TODO(rhcarvalho): check syntax of other important entrypoint playbooks
   - ansible-playbook --syntax-check playbooks/byo/config.yml
-  - tox
-  - cd utils && tox
+  - if [[ "${TEST_DIR}" == "root" ]]; then tox; fi
+  - if [[ "${TEST_DIR}" == "utils" ]]; then cd utils && tox; fi
```

Comment:
We probably need to work out exactly how to properly test utils vs the rest of the repo. I agree that it feels like utils wants to be it's own repo, and it probably should be at some point.

@tbielawa would be interested in your thoughts on this...  Not really sure if we want to lump utils tests back in with the repo root tests, but not quite sure of a better way to make sure we are testing both properly through travis.